stdlib/database/sql/src/main/ballerina/src/sql/client.bal

```
@@ -45,12 +45,14 @@ public type Client abstract client object {
 };
 
 function closedStreamInvocationError() returns Error {
-    return ApplicationError("Stream is closed. Therefore, no operations are allowed " +
-    "further on the stream.");
+    ApplicationError e = ApplicationError("Stream is closed. Therefore, no operations are allowed further" +
+    " on the stream.");
+    return e;
```

Can we return the error directly?
```
return  ApplicationError("...");
```